Multisig/UI/Safe Management/Create Safe/Setup Safe/SelectAddressViewController.swift

```diff
@@ -0,0 +1,192 @@
+//
+//  SelectAddressViewController.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 28.02.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Web3
+
+class SelectAddressViewController: UIAlertController {
+    var completion: (Address) -> Void = { _ in }
+    var chain: Chain?
+    weak var presenter: UIViewController?
+
+    convenience init(chain: Chain?, presenter: UIViewController, completion: @escaping (Address) -> Void) {
+        self.init(title: nil, message: nil, preferredStyle: .actionSheet)
+        self.presenter = presenter
+        self.chain = chain
+        self.completion = completion
+
+        addAction(UIAlertAction(title: "Paste from Clipboard", style: .default, handler: handlePaste(_:)))
+        addAction(UIAlertAction(title: "Scan QR Code", style: .default, handler: handleScan(_:)))
+        addAction(UIAlertAction(title: "Address Book", style: .default, handler: handleAddressBook(_:)))
+        addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: nil))
```

Comment:
I would create another ticket to support ENS & unstoppable domains, this way we can reuse this component in different places in the app WDYT?

Reply:
yes it would evolve into a thing that can be reused in multiple places, but not in this pr